pkg/clusters/addons/kong/kong.go

```diff
@@ -181,6 +235,16 @@ func (a *Addon) Delete(ctx context.Context, cluster clusters.Cluster) error {
 		return fmt.Errorf("%s: %w", stderr.String(), err)
 	}
 
+	if a.license != "" && a.license == KongEnterpriseTestLicense {
+		stderr := new(bytes.Buffer)
+		cmd = exec.Command("kubectl", "delete", "secret", a.license, "--namespace", a.namespace) //nolint:gosec
+		cmd.Stdout = io.Discard
+		cmd.Stderr = stderr
+		if err := cmd.Run(); err != nil {
+			return fmt.Errorf("delete secret err msg %s: %w", stderr.String(), err)
+		}
+	}
+
```

Comment:
This `kubectl` callout includes a bug that probably wont affect CI, but might affect users because it's missing a specific `--kubeconfig` declaration.

You could add that and it would solve the problem, but you could also just use the provided `cluster.Client()` here to delete the secret without having to make a call out to the shell.

Reply:
Fixed https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R264